api/v3/openapi.yaml

```diff
@@ -1618,6 +1618,17 @@ components:
         application/json:
           schema:
             $ref: '#/components/schemas/CreateAclRequestData'
+          example:
+            kind: 'KafkaAcl'
+            metadata:
+              self: 'http://localhost:9391/v3/clusters/cluster-1/acls?resource_type=CLUSTER&resource_name=cluster-1&pattern_type=LITERAL&principal=bob&host=*&operation=DESCRIBE&permission=DENY'
+            resource_type: 'CLUSTER'
+            resource_name: 'cluster-1'
+            pattern_type: 'LITERAL'
+            principal: 'alice'
+            host: '*'
+            operation: 'DESCRIBE'
+            permission: 'DENY'
```

Comment:
@rigelbm want to be sure the values here look good. I have obtained this from the `DeleteAclResponse`.

Reply:
LGTM